test/helpers/governance.js

```diff
@@ -196,6 +197,44 @@ class GovernorHelper {
   }
 }
 
+/**
+ * Encodes a list ProposalStates into a bytes32 representation where each bit enabled corresponds to
+ * the underlying position in the `ProposalState` enum. For example:
+ *
+ * 0x000...10000
+ *   ^^^^^^------ ...
+ *         ^----- Succeeded
+ *          ^---- Defeated
+ *           ^--- Canceled
+ *            ^-- Active
+ *             ^- Pending
+ */
+function proposalStatesToBitMap(proposalStates, options = {}) {
+  if (!Array.isArray(proposalStates)) {
+    expect.fail(`Expected ProposalStates array but found ${typeof proposalStates} instead`);
+  }
```

Comment:
What about

```suggestion
  if (!Array.isArray(proposalStates)) {
    proposalStates = [proposalState];
  }
```

So that you can call with "single state" without using an array